### ISSUE TITLE
How to access console do download additional files of nltk

### ISSUE THREAD

**Issue by michelpf** [Azure/azure-functions-core-tools#1029]
To use NLTK I need do download some files before using.
I set in requirements the library nltk.

```
azure-functions==1.0.0b3
azure-functions-worker==1.0.0b3
grpcio==1.14.2
grpcio-tools==1.14.2
protobuf==3.6.1
six==1.12.0
nltk==3.4
```

Usually we need to download some files before use, for example:

```
import nltk
nltk.download('stopwords')
```

With the new Functions 2.0 I don't have access do Kudu or console in order to download.
There is another way to get access? Or another strategy do download this kind of packets?

**Comment by ankitkumarr** [comment]
@michelpf, I believe it should work as expected if you simply add those two lines at the top of your function (`function_name/__init__.py`)
```
import nltk
nltk.download('stopwords')
```
Please let me know if that does not work for you.

**Comment by michelpf** [comment]
For every api call it will check this files. It is not what I really want.
I'll try to change the download folder to the local folder of function to see if works.

**Comment by ankitkumarr** [comment]
@michelpf, I believe that depends on the plan you are using. 

For consumption plan, yes, depending on the gap between calls (cold start), it may download on every API call. 

I can't imagine a way around it yet, unless those files are somehow packaged with the app. 

@asavaritayal can confirm this.

**Comment by ahmelsayed** [comment]
are these large downloads?

**Comment by craigeddy** [comment]
I have the same question. Not only for `nltk`, but also attempting to use `ftfy` in my function.

Running from VS Code works just fine. However, after deploying to Azure and attempting to run, I get 500 errors and the following in the logs:

```
Exception: FileNotFoundError: [Errno 2] No such file or directory: '/home/site/wwwroot/worker-bundle/ftfy/char_classes.dat'
```
Is there a way to customize `python_docker_build.sh` so that we could add lines such as the following (which is from a dockerfile that I have used to containterize my python app):

```
RUN [ "python", "-c", "import nltk; nltk.download('stopwords')" ]
```

**Comment by michelpf** [comment]
After the tests, I was able to install additional files of NLTK in current function folder:

For download, I used my virtual environment and send the commands in console, not in the function. We need just to download the files, so, start python and then:
```
import nltk
nltk.download('stopwords', download_dir='')
```
After that, at the begining of my function (`__init__.py`), include:

```
import nltk
nltk.data.path.append("/")
```
Local tests and production tests were fine.

Thanks all for supporting me.
